Pcap++/src/PcapLiveDevice.cpp

```diff
@@ -1168,6 +1168,31 @@ void PcapLiveDevice::setDefaultGateway()
 #endif
 }
 
+std::vector<IPAddress> PcapLiveDevice::getIPAddresses() const
+{
+	std::vector<IPAddress> results;
+	for (const auto& address : m_Addresses)
+	{
+		in_addr* ipv4Addr = internal::try_sockaddr2in_addr(address.addr);
+		if (ipv4Addr != nullptr)
+		{
+			results.push_back(IPv4Address(ipv4Addr->s_addr));
+			PCPP_LOG_DEBUG("IPv4 address found: " << results.back());
```

Comment:
Do we need this log?

Reply:
Not sure. Added it for consistency as both `getIPv4Address` and `getIPv6Address` output address logs.

Reply:
This is very old code, I think we can remove the logs 🙂

Reply:
Removed 4100859